monitorrent/plugin_managers.py

```diff
@@ -138,14 +140,14 @@ def get_watching_topics(self):
         return watching_topics
 
     def execute(self, engine):
-        for name, tracker in self.trackers.iteritems():
+        for name, tracker in self.trackers.items():
```

Comment:
Line 89 has list() of items, while line 143 doesn't.
May be a bug in conversion tool?